modules/nf-core/gatk4/haplotypecaller/main.nf

```diff
@@ -1,11 +1,11 @@
 process GATK4_HAPLOTYPECALLER {
     tag "$meta.id"
-    label 'process_medium'
+    label 'process_low'
```

Comment:
have all these been tested? Let's run a few different full size tests before release to check if this interfers with runtime at all

Reply:
We've been running this in prod for a while now without seeing any adverse effects. Haplotypecaller itself is very much single threaded. The extra resources are only useful for decompression and compression of in/and output